packages/generator-helper/src/__tests__/exiting-executable

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env node
 
-require('ts-node/register')
+require('@swc-node/register')
 
-const { generatorHandler } = require('../generatorHandler')
+const { generatorHandler } = require('../generatorHandler.ts')
```

Comment:
This one needs the extension to be specified in order to work, and that does not influence the test in any way.